CMakeLists.txt

```diff
@@ -58,6 +58,10 @@ endif()
 # else will be ignored. PlayRho only supports Debug and Release
 set(CMAKE_CONFIGURATION_TYPES "Debug;Release;" CACHE STRING "Semicolon separated list of supported configuration types. PlayRho only supports Debug and Release configurations.")
 
+# Options for robustness: Useful during development
+# Note: -Wno-error=float-equal should be resolved
+# add_definitions("-Wall -Wextra -Werror -Wfloat-equal -Wundef -Wcast-align -Wwrite-strings -Wunreachable-code -Wconversion -Wno-error=maybe-uninitialized -Wno-error=float-equal -Wno-error=ignored-qualifiers")
```

Comment:
Also, I'll need to play with these flags some myself. I believe I have `-Wall` and `-Wextra` enabled for clang, and things compile without warnings for me. I'd like that to be minimally the case for GCC as well but I see now these flags are not enabled for those nor MSVC via CMake. That's something I'd really like to be changed. I mean the code I'd want to compile on clang, gcc, and MSVC with warnings enabled without emitting warnings and it seems that's not something I got into CMake and CI to enforce ; contrary to my intent, sigh.